rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/rest/RestHandler.java

```diff
@@ -70,7 +74,13 @@ public RestHandler(AbstractConnectController connectController) {
         app.get("/getConfigInfo", this::getConfigInfo);
         app.get("/getAllocatedConnectors", this::getAllocatedConnectors);
         app.get("/getAllocatedTasks", this::getAllocatedTasks);
-        app.get("/plugin/reload", this::reloadPlugins);
+
```

Comment:
An additional suggestion, now all rest APIs of connect do not have a unified return format, which brings certain difficulties to the integration of external systems. I wonder if you are willing to unify the request and response format, which can bring great benefits to developers and users.

Reply:
> An additional suggestion, now all rest APIs of connect do not have a unified return format, which brings certain difficulties to the integration of external systems. I wonder if you are willing to unify the request and response format, which can bring great benefits to developers and users.

Will be processed in  [ISSUE #238]